aws-lambda-java-serialization/src/main/java/com/amazonaws/services/lambda/serialization/JacksonPojoSerializer.java

```diff
@@ -158,16 +161,16 @@ private static JsonFactory createJsonFactory() {
                 .enable(JsonReadFeature.ALLOW_UNQUOTED_FIELD_NAMES)
 
                 //Json Read disabled
-                .disable(JsonReadFeature.ALLOW_JAVA_COMMENTS)			            //this is default as of 2.12.3
-                .disable(JsonReadFeature.ALLOW_YAML_COMMENTS)			            //this is default as of 2.12.3
+                .disable(JsonReadFeature.ALLOW_JAVA_COMMENTS)                               // this is default as of 2.13.4.2
```

Comment:
`this is default` changes look confusing. Starting from which version it started to be default?

Reply:
You are right. Updated :)